BFS_Connected_Components/Q11_most_stones_removed.py

```diff
@@ -0,0 +1,40 @@
+import collections
+
+def build_graph(stones):
+    adj_list = [[] for _ in range(len(stones))]
+    
+    for index1, stone1 in enumerate(stones):
+        for index2, stone2 in enumerate(stones):
+            if index1 == index2:continue
+            if stone1[0] == stone2[0] or stone1[1] == stone2[1]:
+                adj_list[index1].append(index2)
+                adj_list[index2].append(index1)
+    return adj_list
+
+def removeStones(stones):
+    adj_list = build_graph(stones)          
+    ROWS, COLS = len(stones), len(stones[0])
+    seen = set()
+    
+    def get_count_of_removable_stones(stone):
+        q = collections.deque([stone])
+        seen.add(stone)
+        count = 0
+        while q:
+            stone_new = q.popleft()
+            count += 1
+            for neighbor in adj_list[stone_new]:
+                if neighbor in seen:continue
+                q.append(neighbor)
+                seen.add(neighbor)
+                    
+        return count - 1
```

Comment:
This looks arbitrary. You want your code to be self-documenting. It should describe what it is doing. Something like ```
num_removable_stones = count - 1
return num_removable_stones
```
would be clearer